stdlib/public/SwiftShims/System.h

```diff
@@ -69,6 +69,16 @@
 // Only the bottom 56 bits are used, and heap objects are eight-byte-aligned.
 #define SWIFT_ABI_DEFAULT_64BIT_SPARE_BITS_MASK 0xFF00000000000007ULL
 
+// Poison sentinel value recognized by LLDB as a former reference to a
+// potentially deinitialized object. It uses no spare bits and cannot point to
+// readable memory.
+//
+// This is not ABI per-se but does stay in-sync with LLDB. If it becomes
+// out-of-sync, then users won't see a friendly diagnostic when inspecting
+// references past their lifetime.
+#define SWIFT_ABI_DEFAULT_REFERENCE_POISON_DEBUG_VALUE_32 0x00000880ULL
+#define SWIFT_ABI_DEFAULT_REFERENCE_POISON_DEBUG_VALUE_64 0x0000000000000880ULL
```

Comment:
Just curious, since there's one value, why are there two constants?

Reply:
Thanks, they were supposed to have a different type suffix. I fixed it here:
https://github.com/apple/swift/pull/36389/commits/02e9a612ba5f3d810cdd78dceaf2a50863e1aab5

I'm not sure we need two constants except that we seem to have a convention that these constants aren't magically zero-extended.